<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFPFHFeature</class>
 <widget class="QSubDialogBase" name="QFPFHFeature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FPFH特征</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>邻域半径（double）</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>邻域数量（int）</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>形状：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="10" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>待识别对象</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>262</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QRendView" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QRendView</class>
   <extends>QWidget</extends>
   <header>QRendView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSubDialogBase</class>
   <extends>QWidget</extends>
   <header>qsubdialogbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFPFHFeature</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApply()</slot>
 </slots>
</ui>
